src/idsk/components/header-web/template.njk

```diff
@@ -0,0 +1,315 @@
+                                                        {% else %}
+                                                                SK <button class="idsk-header-web__brand-gestor-button">e-Gov<div class="idsk-header-web__link-arrow"></div></button>
+                                                        {% endif %}   
+                                                </span>
+                                        </div>
+                                        
+                                        <div class="idsk-header-web__brand-language">
+                                                <button class="idsk-header-web__brand-language-button">{{ currentLanguage if currentLanguage else 'Slovenčina' }}<div class="idsk-header-web__link-arrow"></div></button> 
+                                                <ul class="idsk-header-web__brand-language-list">
+                                                        {% for language in brand.languages | sort(attribute='text') %}
+                                                        <li class="idsk-header-web__brand-language-list-item">
+                                                                <a class="govuk-link idsk-header-web__brand-language-list-item-link" title="{{ language.text }}" href="{{ language.link }}">
+                                                                        {{ language.text }}
+                                                                </a>
+                                                        </li>
+                                                        {% endfor %}
+                                                </ul>  
+                                        </div>
+                                </div>        
+
+                        </div>
+                        <div class="idsk-header-web__brand-dropdown">
+                                <div class="govuk-grid-row">
+                                        <div class="govuk-grid-column-one-half">
+                                                <p class="govuk-body-s">Doména Gov.sk je oficálna</p>
+                                                <p class="govuk-body-s">Slovak government websites always use a .gov.sk domain. Before sharing sensitive information online, make sure.</p>
+                                        </div>
+                                        <div class="govuk-grid-column-one-half">
+                                                <p class="govuk-body-s">Táto stránka je zabezpečená</p>
+                                                <p class="govuk-body-s">Slovak government websites always use a .gov.sk domain. Before sharing sensitive information online, make sure.</p>
+                                        </div>
+                                </div>  
+                        </div>
+                        
+                </div>
+                
```

Comment:
nerozumiem tomu celkom preco su tu nove riadky, necleni mi to nijak bloky ani nic